src/main/java/gregtech/api/unification/material/materials/FirstDegreeMaterials.java

```diff
@@ -0,0 +1,1227 @@
+                .ingot().fluid()
+                .color(0x80808C).iconSet(SHINY)
+                .flags(STD_METAL, GENERATE_FOIL, GENERATE_ROD)
+                .components(Vanadium, 3, Gallium, 1)
+                .cableProperties(GTValues.V[6], 4, 2)
+                .blastTemp(4500)
+                .build();
+
+        WroughtIron = new Material.Builder(335, "wrought_iron")
+                .ingot().fluid()
+                .color(0xC8B4B4).iconSet(METALLIC)
+                .flags(EXT2_METAL, MORTAR_GRINDABLE, GENERATE_RING, GENERATE_LONG_ROD, DISABLE_DECOMPOSITION, BLAST_FURNACE_CALCITE_TRIPLE)
+                .components(Iron, 1)
+                .toolStats(6.0f, 3.5f, 384)
+                .fluidPipeProperties(2387, 30, true)
+                .build();
+        Iron.getProperty(PropertyKey.INGOT).setSmeltingInto(WroughtIron);
+
+        Wulfenite = new Material.Builder(336, "wulfenite")
+                .dust(3).ore()
+                .color(0xFF8000)
+                .components(Lead, 1, Molybdenum, 1, Oxygen, 4)
+                .build();
+
+        YellowLimonite = new Material.Builder(337, "yellow_limonite")
+                .dust().ore()
+                .color(0xC8C800).iconSet(METALLIC)
+                .flags(DECOMPOSITION_BY_CENTRIFUGING, BLAST_FURNACE_CALCITE_DOUBLE)
+                .components(Iron, 1, Hydrogen, 1, Oxygen, 2)
+                .build();
+
+        YttriumBariumCuprate = new Material.Builder(338, "yttrium_barium_cuprate")
+                .ingot().fluid()
+                .color(0x504046).iconSet(METALLIC)
+                .flags(EXT_METAL, GENERATE_FINE_WIRE)
+                .components(Yttrium, 1, Barium, 2, Copper, 3, Oxygen, 7)
+                .cableProperties(GTValues.V[6], 4, 4)
+                .build();
+
+        NetherQuartz = new Material.Builder(339, "nether_quartz")
+                .gem(1).ore(2, 1)
+                .color(0xE6D2D2).iconSet(QUARTZ)
+                .flags(STD_SOLID, NO_SMELTING, CRYSTALLIZABLE, EXCLUDE_BLOCK_CRAFTING_BY_HAND_RECIPES, DISABLE_DECOMPOSITION)
+                .components(Silicon, 1, Oxygen, 2)
+                .build();
+
+        CertusQuartz = new Material.Builder(214, "certus_quartz")
+                .gem(1).ore(2, 1)
+                .color(0xD2D2E6).iconSet(QUARTZ)
+                .flags(STD_SOLID, NO_SMELTING, CRYSTALLIZABLE, DISABLE_DECOMPOSITION)
+                .components(Silicon, 1, Oxygen, 2)
+                .build();
+
+        Quartzite = new Material.Builder(340, "quartzite")
+                .gem(1).ore(2, 1)
+                .color(0xD2E6D2).iconSet(QUARTZ)
+                .flags(NO_SMELTING, CRYSTALLIZABLE, DISABLE_DECOMPOSITION, GENERATE_PLATE)
+                .components(Silicon, 1, Oxygen, 2)
+                .build();
+
+        Graphite = new Material.Builder(341, "graphite")
+                .ingot().ore().fluid()
+                .color(0x808080)
+                .flags(STD_METAL, NO_SMELTING, FLAMMABLE, DISABLE_DECOMPOSITION)
```

Comment:
Again, this material still has a decomposition recipe, so I am unsure if this flag is working as expected

Reply:
I am relatively sure I manually added this recipe elsewhere.